src/guides/config.md

```diff
@@ -174,7 +174,20 @@ $ HSD_NETWORK=testnet HSD_HTTP_HOST=0.0.0.0 HSD_WALLET_HTTP_HOST=0.0.0.0 HSD_WAL
 - `wallet-auth`: Enable token auth for wallets (default: false).
 - `admin-token`: Token required if `wallet-auth` is enabled: restricts access to [all wallet admin routes.](https://handshake-org.github.io/api-docs/#wallet-admin-commands)
 
+### Example configuration file
+
+The following file could be placed at `~/.hsd/hsd.conf` for use with a mainnet hsd node.
+All the settings in the example below are defaults, meaning you do NOT need to use
+this `hsd.conf` if you do not need to change any of the values. It is
+provided only as an example of the configuration file syntax.
```

Comment:
Configuration options are newline delimited. The key value pairs are delimited by a colon. All configuration can be set via command line, environment variable or the config file.